database/schema.sql

```diff
@@ -0,0 +1,34 @@
+CREATE TABLE IF NOT EXISTS 'author' (
+  'id' integer PRIMARY KEY AUTOINCREMENT,
+  'first_name' text NOT NULL,
+  'last_name' text NOT NULL,
+);
+
+CREATE TABLE IF NOT EXISTS 'game' (
+  'id' integer PRIMARY KEY AUTOINCREMENT,
+  'multiplayer' text NOT NULL,
+  'last_played_at' date NOT NULL,
+  'item_id' integer REFERENCES 'item'
+);
+
+CREATE TABLE item (
+  id INT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+  publish_date DATE NOT NULL,
+  archived BOOLEAN NOT NULL,
+  label_id INT NOT NULL REFERENCES label(id) ON DELETE CASCADE,
+  author_id INT NOT NULL REFERENCES author(id)  ON DELETE CASCADE,
+  genre_id INT NOT NULL REFERENCES genre(id) ON DELETE CASCADE  
+);
+
+CREATE TABLE book (
+  id INT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY ,
+  publisher VARCHAR(255) NOT NULL,
+  cover_state VARCHAR(25) NOT NULL,  
+  CONSTRAINT fk_item FOREIGN KEY (item_id) REFERENCES item(id) ON DELETE CASCADE
+);
+
+CREATE TABLE label (
```

Comment:
![image](https://user-images.githubusercontent.com/30554633/222410061-8f76114b-29b8-4deb-b9a8-f9b2719fa651.png)

Reply:
Hello Asha, I get the following errors when I run the schema codes.